### PR TITLE
feat(i18n): add Malay(ms) translation

### PR DESCRIPTION
**Pull request by hcw2175**
add locale: ms

### CODE REVIEW

File: packages/locale/lang/ms.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      titles: ['Senarai 1', 'Senarai 2'],
+      filterPlaceholder: 'Masukkan kata kunci',
+      noCheckedFormat: '{total} item', // to be translated
+      hasCheckedFormat: '{checked}/{total} diperiksa',
+    },
+    image: {
+      error: 'FAILED',
+    },
+    pageHeader: {
+      title: 'Belakang', // to be translated
```
Comment (plainheart) [inline packages/locale/lang/ms.ts:172]: Please remove the comment `// to be translated` if one item has been translated.

File: packages/locale/lang/ms.ts [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    },
+    tree: {
+      emptyText: 'Tiada Data',
+    },
+    transfer: {
+      noMatch: 'Tiada data yang sepadan',
+      noData: 'Tiada Data',
+      titles: ['Senarai 1', 'Senarai 2'],
+      filterPlaceholder: 'Masukkan kata kunci',
+      noCheckedFormat: '{total} item', // to be translated
```
Comment (plainheart) [inline packages/locale/lang/ms.ts:165]: Can you also translate the rest of the items?

**Review by plainheart — CHANGES_REQUESTED** [review]

File: packages/locale/lang/ms.ts (hunk 1 above)
Comment (hcw2175) [reply packages/locale/lang/ms.ts]: Have been removed

File: packages/locale/lang/ms.ts (hunk 2 above)
Comment (hcw2175) [reply packages/locale/lang/ms.ts]: Have been modified

**Review by plainheart — APPROVED** [review]

**Comment by warmthsea** [comment]
https://github.com/element-plus/element-plus/blob/dev/docs/en-US/guide/i18n.md
Can you add this new language to this file?

**Pull request merged**